src/components/button/primary/button-primary.tsx

```diff
@@ -0,0 +1,14 @@
+import React, {FunctionComponent} from 'react';
+import './button-primary.scoped.scss';
+
+export type ButtonPrimaryProps = Partial<{
+    text: string;
+}>;
+
+export const ButtonPrimary: FunctionComponent<ButtonPrimaryProps | undefined> = (
```

Comment:
- [ ] All our design system component should use the `Toniq` prefix

Reply:
resolved